packages/cli/src/bin/helpers.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+
+export default function lookForRootDirectory(dirPath: string): string | null {
+  if (dirPath.length === 0) return null;
+  const dirFiles = fs.readdirSync(dirPath);
+
+  if ((dirFiles.includes('package.json') && dirFiles.includes('package-lock.json')) || dirFiles.includes('zos.json')) {
+    return dirPath;
+  } else {
+    const dir = dirPath.split('/');
```

Comment:
Not every OS use '/' as path separator. User [path.sep](https://nodejs.org/api/path.html#path_path_sep) instead.

Reply:
Good catch!